docs/learn/calls.md

```diff
@@ -132,7 +139,7 @@ For all providers, we have only included additional call parameters that are not
 
```

Comment:
nit: the api documentation above is missing links to the new providers we added (which also likely means that the providers call params are missing the learn docs link in their docstrings)